examples/catvdog/README.rst

```diff
@@ -0,0 +1,52 @@
+Cat and Dog Image Classifier
+============================
+
+Requirements
+------------
+
+* Follow the documented `installation instructions <https://klio.readthedocs.io/en/latest/quickstart/installation.html>`_.
+* Python 3.6+
+
+.. note::
+
+    Make sure the minor version matches with the base Docker image stated on the ``FROM`` line in the `Dockerfile <./Dockerfile#L2>`_.
```

Comment:
Is this referring to the minor version on the user's local dev environment, or something else?

Reply:
Yeah - is there a way that I could make that a bit more clear?

Reply:
Actually that raises another question...is that requirement actually necessary?  Isn't the reason we have distinct `klio-cli` and `klio-exec` so that the minor python versions can differ?

Reply:
actually yeah you're right lol will update